.gitignore

```diff
@@ -1 +1,9 @@
 /.idea/
+/Hello_Coffee/Hello_Coffee.iml
```

Comment:
@AraChilingaryan you can write *.iml instead of excluding each seperatly

Reply:
okay next time i will do that

Reply:
done